cypress/support/pages/profile.pageObgect.js

```diff
@@ -0,0 +1,21 @@
+import PageObject from '../PageObject';
```

Comment:
Rename the file from 'profile.pageObgect.js' to 'profile.pageObject.js' to match the correct naming convention.